ethcontract-mock/src/lib.rs

```diff
@@ -232,6 +232,7 @@ pub use ethcontract::contract::Signature;
 mod details;
 mod predicate;
 mod range;
+pub mod utils;
```

Comment:
Should this maybe be behind a feature?

Reply:
I think it doesn't add too much code size or complexity to warrant a feature. On the other hand, it may grow at some point. I'm not sure about this.

Reply:
Lets revisit the feature idea if it adds specific crate dependencies.